Source/Common/gdcmSwapper.txx

```diff
@@ -135,7 +135,7 @@ namespace gdcm
 
   template <> inline Tag SwapperNoOp::Swap<Tag>(Tag val)
     {
-    return Tag( Swap(val.GetGroup()), Swap(val.GetElement()) );
+    return Tag( Swap((uint32_t)val.GetElementTag()) );
```

Comment:
Same comment, refer to 874fa717ac

Reply:
Same. Compare:

https://github.com/malaterre/GDCM/blob/008e437d6ea81979bfa23301ae6e62f82272fe64/Source/Common/gdcmSwapper.txx#L214

https://github.com/malaterre/GDCM/blob/008e437d6ea81979bfa23301ae6e62f82272fe64/Source/Common/gdcmSwapper.txx#L138

Why are they different?

Reply:
So in fact in 874fa717ac769ecf7a43c189a7974b1eb100aade looks like you changed only the big endian branch, and not the little endian branch.